.github/workflows/stale.yml

```diff
@@ -0,0 +1,70 @@
+name: Stale issues and pull-requests
+on:
+  schedule:
+    - cron: 0 * * * *
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4.0.0
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+          # Idle number of days before marking issues/prs stale.
+          days-before-stale: 1000
+
+          # Idle number of days before closing stale issues/prs.
+          days-before-close: 28
+
+          # Comment on the staled issues.
+          stale-issue-message: |
+            In order to maintain a backlog of relevant issues, we automatically label them as stale after 1,000 days of inactivity.
+
+            If this issue is still important to you, then please comment on this issue or remove the stale label.
+
+            Otherwise this issue will be automatically closed in 28 days time.
+
+          # Comment on the staled prs.
+          stale-pr-message: | 
+            In order to maintain a backlog of relevant PRs, we automatically label them as stale after 1,000 days of inactivity.
+
+            If this PR is still important to you, then please comment on this PR or remove the stale label.
+
+            Otherwise this PR will be automatically closed in 28 days time.
+
+          # Comment on the staled issues while closed.
+          close-issue-message: |
+            This stale issue has been automatically closed due to a lack of community activity.
+            
+            If you still care about this issue, then please either:
+              * Re-open this issue, if you have sufficient permissions, or
+              * Add a comment pinging `@SciTools/iris-devs` who will re-open on your behalf.
+
+          # Comment on the staled prs while closed.
+          close-pr-message: |
+            This stale PR has been automatically closed due to a lack of community activity.
+            
+            If you still care about this PR, then please either:
+              * Re-open this PR, if you have sufficient permissions, or
+              * Add a comment pinging `@SciTools/iris-devs` who will re-open on your behalf.
+
+          # Label to apply on staled issues. 
+          stale-issue-label: Stale
+
+          # Label to apply on staled prs.
+          stale-pr-label: Stale
+
+          # Labels on issues exempted from stale.
+          exempt-issue-labels: |
+            "Status: Blocked,Status: Decision Required,Status: Needs Info,Status: Stalled,Peloton 🚴‍️"
```

Comment:
I would argue that "Needs Info" and "Stalled" should not be in this list.  They should be staled out if they go 1000 days without info or unstalling

Reply:
Cool, I'd go with that... did I miss out any labels that you'd consider should be in the list?

Reply:
Done :+1: